File: LeetCode-Programs/Median of Two Sorted Arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        
        double median;
        vector<double> vec;
        for(int i=0;i<nums1.size();i++)
        {
            vec.push_back(nums1[i]);
        }
        for(int i=0;i<nums2.size();i++)
        {
            vec.push_back(nums2[i]);
        }
        sort(vec.begin(), vec.end());
        int s=vec.size();
        
        int mid=(0+s-1)/2;
        if(s%2==0)
            median=(vec[mid]+vec[mid+1])/2;
        else
            median=vec[mid];
        return median;
    }
};

File: LeetCode-Programs/Median of Two Sorted Arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:

        num=[]
        for i in range(len(nums1)):
            num.append(nums1[i])
        for i in range(len(nums2)):
            num.append(nums2[i])
        num.sort()
        s=len(num)
        
        mid=(0+s-1)//2
        if s%2==0:
            median=(num[mid]+num[mid+1])/2
        else:
            median=num[mid]
        return median
